browser/ui/brave_actions/brave_action_view_controller.cc

```diff
@@ -122,6 +128,7 @@ void BraveActionViewController::OnPopupClosed() {
   popup_host_observation_.Reset();
   popup_host_ = nullptr;
   view_delegate_->OnPopupClosed();
+  actions_container_delegate_->OnPopupClosed(extension()->id());
```

Comment:
this doesn't seem right to me unless this code was previously broken cc @petemill

Reply:
This change was done basing on comment from here:
https://github.com/brave/brave-core/blob/4384e9425c5b56953f68469f16d224e1aba98859/browser/ui/views/brave_actions/brave_actions_container.cc#L209